staging-web/config.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
```

Comment:
configuration should not be python but toml or similar
Also this should be option to deploy, not part of the main deploy system. would be better to have is own repository.